### DISCUSSION TITLE
Solar Fusion oauth2 and html calls

### DISCUSSION THREAD

**Discussion by Alpha-9-Sierra** [tijsverkoyen/HomeAssistant-FusionSolar#147] [Q&A]
Hello, I'd like to create an Alexa skill using oauth2.0, I created a Huawei dev account and setup huawei ID on Alexa but I don't know how to create html calls for the skill because there's no documentation available. Is there a solution to implement fusion solar oauth and is there documentation available somewhere ?

Regards,

**Comment by tijsverkoyen** [comment]
@TboneBEL just to be sure: I'm not related to Fusion Solar or Huawei at all. This integration is something I created for myself to see data from my solar panels in Home Assistant. 

I have use https://support.huawei.com/enterprise/en/doc/EDOC1100261860/9e1a18d2/login-interface to create this Home Assistant. Home Assistant provides a lot of infrastructure to create calls, update entities, ... So I don't know about Alexa Skill.

Maybe you can check out https://www.home-assistant.io/integrations/alexa.smart_home/
